frontend/pages/index.vue

```diff
@@ -41,16 +41,21 @@
                     <v-col cols="12" sm="3" md="3">
                       <v-text-field
                         v-model="collectionId"
-                        :rules="collectionRules"
+                        :rules="[v => !!v || 'Collection ID is required']"
+                        :items="collectionRules"
                         label="Collection ID"
+                        class="collection-id-error-text"
                         hint="e.g. col12345"
                         required
                       />
                     </v-col>
                     <v-col cols="12" sm="3" md="3">
                       <v-text-field
                         v-model="version"
+                        :rules="[v => !!v || '']"
+                        :items="versionItems"
```

Comment:
versionItems should not be added because we don't have a list of versions to choose from currently.

If rules are added here then it's not allowed to have an empty version box. My assumption was that empty version editbox=latest version. If that should be changed then an error text should be added instead of `''`

Reply:
@therealmarv I removed the added lines for version. I will only test colid, style and server error messages.